pom.xml

```diff
@@ -47,6 +47,12 @@
 				<artifactId>kafka-clients</artifactId>
 				<version>${kafka.version}</version>
 			</dependency>
+			<dependency>
+				<groupId>org.apache.kafka</groupId>
+				<artifactId>kafka-clients</artifactId>
+				<version>${kafka.version}</version>
+				<classifier>test</classifier>
+			</dependency>
```

Comment:
I think this just needs to be in test scope; looks like a bug in our docs.

Reply:
will address this.